lyse/__init__.py

```diff
@@ -405,7 +409,13 @@ def get_trace(self,name):
             if not name in h5_file['data']['traces']:
                 raise Exception('The trace \'%s\' does not exist'%name)
             trace = h5_file['data']['traces'][name]
-            return array(trace['t'],dtype=float),array(trace['values'],dtype=float)         
+            
+            if raw_data:
+                data = trace[:]
```

Comment:
Could we change this to the more general (but less intuitive looking) `data = trace[()]`?